src/index.ts

```diff
@@ -839,6 +837,13 @@ export default function autoAnimate(
     disable: () => {
       enabled.delete(el)
     },
+    destroy() {
+      parents.delete(this.parent);
+      mutations?.disconnect();
+      resize?.disconnect();
```

Comment:
We don’t want to disconnect the mutation observer or the resize observer. These are global instantiations (1 per load). Destroying them would destroy all other observers on the page. `parents` is the only non-weak set/map here so we should keep that one.